src/content/changes/cluster-apps-for-aws/cluster-aws/v0.76.0-alpha.1.md

```diff
@@ -0,0 +1,18 @@
+---
```

Comment:
@pipo02mix before approving – I asked in [chat](https://gigantic.slack.com/archives/C0559SH3RJ4/p1715762256942609) about including this alpha release tag or not